jasmine/spec/invertedIndexTest.js

```diff
@@ -1,55 +1,96 @@
-/* Test Setup */
-const myInvertedIndex = new InvertedIndexClass();
-const book = require('./../books.json');
-const emptyBook = require('./../bookempty.json');
+/* global InvertedIndexClass */
+import book from './../book.json';
+import bookempty from './../bookempty.json';
+import bookwrongformat from './../bookwrongformat.json';
+import notBook from './../notBook.json';
+import books3 from './../books3.json';
 
+const myInvertedIndex = new InvertedIndexClass();
 myInvertedIndex.files['book.json'] = book;
-
+myInvertedIndex.files['books3.json'] = books3;
 
 /* Test Suites */
 describe('Inverted Index Test', () => {
   describe('ReadFile', () => {
-    it('should return false when checking a bad JSON array', () => {
-      expect(myInvertedIndex.readFile(emptyBook))
+    it('should return false when checking an empty JSON array', () => {
+      expect(myInvertedIndex.readFile(bookempty))
         .toBeFalsy();
     });
-
-    it('should return true when if a good JSON array', () => {
+    it('should return false when checking an wrongformatted JSON array', () => {
+      expect(myInvertedIndex.readFile(bookwrongformat))
+        .toBeFalsy();
+    });
+    it('should return false when book is not an array', () => {
+      expect(myInvertedIndex.readFile(notBook))
```

Comment:
It is advisable to use a descriptive variable name when declaring variables
I don't feel that `notBook` is descriptive enough